src/containers/AuctionPriceBar/index.ts

```diff
@@ -4,10 +4,15 @@ import { State } from 'types'
 import AuctionPriceBar from 'components/AuctionPriceBar'
 
 // TODO: create state for lastAuctionPrice for each token
-const mapState = ({ tokenPair }: State) => ({
-  sellToken: tokenPair.sell,
-  sellTokenPrice: '123',
-  buyToken: tokenPair.buy,
-})
+const mapState = (state: State) => {
+  const ratioPairSelector: any = state.ratioPairs.find((pair: any) => 
```

Comment:
this would benefit from a memoized selector. Actually, use `findRatioPair` from `/containers/AuctionSellingGetting`. Move it to `/selectors/ratioPairs.ts`, please.